sw/device/lib/dif/dif_i2c.h

```diff
@@ -419,6 +585,98 @@ OT_WARN_UNUSED_RESULT
 dif_result_t dif_i2c_write_byte(const dif_i2c_t *i2c, uint8_t byte,
                                 dif_i2c_fmt_t code, bool suppress_nak_irq);
 
+/**
+ * Pushes a byte into the TX FIFO to make it available for I2C Read
+ * recieved by the target device.
```

Comment:
host device? (since "target" is the mode our device is in, right?)

Reply:
That could be clearer, I think of Reads as Sent by the host and then responded to, and was trying to refer to the internal behavior (that writing this FIFO would merely prepare for the Read, not actually trigger it) but I'll clean it up.

Reply:
Ah, so maybe "for an I2C Read command received by the provided device"? (as opposed to the data)